### ISSUE TITLE
Impound glitch

### ISSUE THREAD

**Issue by Valkatastz** [Horizahn/hrp_pd_impound#6]
When I wanted to test it out, the menu for impound information to be filled in has glitched and couldn't be closed, confirmed or canceled. The game actually was in frozen state where no keys were working and I had to crash the whole game in order to restart it. If someone has a solution on that, please share it with me. Meanwhile I will try to fix it by my own, sorry if I have to change some bit of the script. 

As I said, if you have any solution to this issue, let me know.

**Comment by YahTheDev** [comment]
Did you restart the resource while having one of the views open? Because you don't want to do that. Its some bug with NUI i think, or my implementation. Havemt had the time to figure it out. 

Otherwise you peobably have either or both:
- wrong resource name
- wrong folder location
- giant errors in the log that should be shared.

**Comment by Valkatastz** [comment]
Hello Horizahn, thank you for your response. I didn't restart any resource when I have opened the form to fill and submit for impounding a car. I have all dependencies (including the esx_migrate). I have plate in owned_vehicles into the DB. Followed the instructions on fivem forum, the folder is in the resource folder, however there is no error to the console. It says "started hrp_pd_impound" but in game when I try to impound a vehicle, that form cannot be closed. The two buttons on bottom don't do anything. I click the impound or cancel button, nothing. Pressing esc button to close it or any other buttons, nothing. No reaction at all from it and I had to crash the game to close it. Also, the buttons didn't do the connection to the database but when I've checked the files, the strings seemed to be okay and didn't throw an error for the database at all which is doubled weird.
